poly-org.el

```diff
@@ -78,10 +81,7 @@ Used in :switch-buffer-functions slot."
   :body-indent-offset 'org-edit-src-content-indentation
   :indent-offset 'org-edit-src-content-indentation)
 
-(define-innermode poly-org-latex-innermode
-  :fallback-mode 'host
-  :head-mode 'host
-  :tail-mode 'host
+(define-innermode poly-org-latex-innermode poly-org-root-innermode
   :mode 'latex-mode
```

Comment:
I am actually no longer sure about this change. It looks like there is little support from org for latex chunks so most of configuration options cannot be reused. Also, shouldn't head be of body type here?

Reply:
In the same fashion as src chunks, latex chunks are typically edited by entering the indirect buffer environment with C-c `, so I thought equivalent treatment would make sense.

> most of configuration options cannot be reused

what kind of configuration options?

> Also, shouldn't head be of body type here?

honestly not sure what each of them entail, but this had worked for me

Reply:
> so I thought equivalent treatment would make sense.

I see. Ok. If latex is a first class environment in org then yes.

>> most of configuration options cannot be reused
> what kind of configuration options?

Object's slots like head-mode, regexp etc. So it doesn't make much sense to inherit the latex from org src. 

I will play with this as soon as I can. I am having limited computer access these weeks.

Reply:
> So it doesn't make much sense to inherit the latex from org src.

I don't think I had made it inherit from the org-src ones.

> I will play with this as soon as I can. I am having limited computer access these weeks.

np, take your time.